apps/studio/src/features/editing-experience/LayoutSelection.tsx

```diff
@@ -0,0 +1,243 @@
+
+const LAYOUT_DATA: {
+  layoutDisplayName: string
+  layoutTypename: LayoutType
+  layoutDescription: string
+  imageSource: string
+}[] = [
+  {
+    layoutDisplayName: 'Default',
+    layoutTypename: 'content',
+    layoutDescription: 'This is the most basic layout for your content.',
+    imageSource: '/assets/layout-card/default_layout_card.png',
+  },
+  {
+    layoutDisplayName: 'Article',
+    layoutTypename: 'article',
+    layoutDescription:
+      'Designed for the perfect reading experience. Use this layout for text-heavy content, such as news, press releases, and speeches',
+    imageSource: '/assets/layout-card/article_layout_card.png',
+  },
+]
+
+function LayoutSelection(props: LayoutSelectionProps): JSX.Element {
+  // need state here to keep track of selected layout
+  const [selectedLayout, setSelectedLayout] = useState(LAYOUT_DATA[0]!)
+  return (
+    <>
+      {/* This is the floating header */}
+      <VStack position="fixed" h="100vh">
+        <VStack p="1.5rem 2rem" w="100vw" gap="0.5rem" alignItems="flex-start">
+          <Button variant="link">
+            <Icon as={BiLeftArrowAlt} />
+            Back To All Pages
+          </Button>
+          <HStack p="0" justifyContent="space-between" w="100%">
+            <Text textStyle="h4" color="base.content.default">
+              {`Choose a layout for "`}
+              {props.pageName}
+              {`"`}
+            </Text>
+            <Button size="xs">
+              Start with this layout
+              <BxRightArrowAlt fontSize="1.5rem" />
+            </Button>
+          </HStack>
+        </VStack>
+        <Grid
+          w="100vw"
+          h="calc(100% - 7rem)"
+          templateColumns="repeat(4, 1fr)"
+          gap={0}
+        >
+          <GridItem w="100%" colSpan={1} bg="grey.50" overflowY="scroll">
+            <VStack p="2rem" gap="2rem" h="fit-content">
+              {/* Picking Layouts */}
+              {LAYOUT_DATA.map((e) => {
+                return (
+                  <VStack
+                    key={e.layoutTypename}
+                    p="0"
+                    gap="0"
+                    cursor="pointer"
+                    onClick={() => {
+                      setSelectedLayout(e)
+                    }}
+                    _hover={
+                      selectedLayout.layoutTypename === e.layoutTypename
+                        ? {}
+                        : {
+                            '.layout-title': { color: 'base.content.brand' },
+                            '.hover-text': {
+                              opacity: 1,
+                            },
+                            '.hover-border': {
+                              borderColor: useToken(
+                                'colors',
+                                'interaction.main-subtle.hover',
+                              ),
+                            },
+                          }
+                    }
```

Comment:
this chunk suffers from a readability issue as well as a locality issue. what i mean is that the changes that we want to apply (`.layout-title` for example) isn't located close to where the changes are required. 

could we locate this where they are required instead?

Reply:
Hmm would [_groupHover](https://v2.chakra-ui.com/docs/styled-system/style-props#pseudo) work in this scenario?

Reply:
Yes, looks like this is the way to do this. replaced with role:group and _groupHover.